bokehjs/src/lib/models/text/math_text.ts

```diff
@@ -187,7 +187,8 @@ export abstract class MathTextView extends BaseTextView implements GraphicsBox {
   size(): Size {
     const {width, height} = this._size()
     const {angle} = this
-    if (!angle)
+
+    if (angle == null || angle == 0 || typeof angle == "undefined")
```

Comment:
Do you need to handle `undefined` here? `angle == null` already handles that. Besides, if you need to distinguish `null` from `undefined`, then you should really use symbols instead.